examples/application_commands/cog_example.py

```diff
@@ -0,0 +1,27 @@
+from nextcord import Interaction, message_command, slash_command, user_command
+from nextcord.ext import commands
+
+
+class ApplicationCommandCog(commands.Cog):  # Subclassing commands.Cog as you always do.
+    def __init__(self, bot):  # takes in bot for the setup later
+        self.bot = bot  # making it global to this class.
+
+    @slash_command(guild_ids=[...])  # isn't really any different to normal commands.
+    async def my_slash_command(self, interaction: Interaction):
+        await interaction.response.send_message("this is a slash command in a cog!")
+
+    @user_command(guild_ids=[...])  # Again very simular to normal commands.
+    async def my_user_command(
+        self, interaction: Interaction, member
+    ):  # passing in member as it is intaked
+        await interaction.response.send_message(f"hello! {member}")
```

Comment:
```suggestion
        await interaction.response.send_message(f"Hello, {member}!")
```
Minor consistency/language change.